plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -266,6 +435,13 @@ export default function Content(): JSX.Element {
 			) }
 
 			{ renderContent() }
+			{ shouldShowLoadMoreButton() && (
```

Comment:
Perhaps,

```suggestion
			{ ! isLoading && shouldShowLoadMoreButton() && (
```

(If you accept the suggestion, don't forget to extract `isLoading` from `marketplaceContextValue`)

I'm seeing this edge-case (you have to be super fast to reproduce it :))


https://github.com/user-attachments/assets/5fdcc3db-ca4f-45a5-8cff-1467e9a2f4b3

Reply:
Ha! I had that check, and removed it after the changes we made in the code chat. Added back in aadaeb3